packages/dd-trace/src/proxy.js

```diff
@@ -25,7 +25,7 @@ class Tracer extends NoopProxy {
     try {
       const config = new Config(options) // TODO: support dynamic config
 
-      if (!config.isCiVisibility) {
+      if (config.remoteConfig?.enabled !== false && !config.isCiVisibility) {
```

Comment:
when `config={}`, `config.remoteConfig?.enabled !== false` would still be `true`. In other words, the remoteConfig is enabled by default unless explicitly turned off.